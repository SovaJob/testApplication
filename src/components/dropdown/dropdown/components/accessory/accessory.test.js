// @flow


/* REACT */
import React from "react";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import Accessory from "./accessory";


describe("Accessory tests", () => {

  test("renders correctly", (): void => {
    const tree = renderer
      .create(
        <Accessory
          baseColor="red"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
