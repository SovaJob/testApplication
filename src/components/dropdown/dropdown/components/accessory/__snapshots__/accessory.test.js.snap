// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accessory tests renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 41,
      "justifyContent": "center",
      "width": 48,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "height": 10.5,
        "overflow": "hidden",
        "width": 24,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "height": 13.5,
            "transform": Array [
              Object {
                "translateY": -7,
              },
              Object {
                "rotate": "45deg",
              },
            ],
            "width": 16,
          },
          Object {
            "backgroundColor": "red",
          },
        ]
      }
    />
  </View>
</View>
`;
